pkg/manifests/manifests_test.go

```diff
@@ -3315,6 +3325,87 @@ func TestTelemeterConfiguration(t *testing.T) {
 	}
 }
 
+func TestTelemeterClientSecret(t *testing.T) {
```

Comment:
I've maintained these tests but with the new e2e tests I'm not so sure it's worth it to test this function so much. Let me know if you want me to remove it

Reply:
It's fine for me to keep the unit tests since they existed before the e2e test.